esmvaltool/recipes/recipe_climwip.yml

```diff
@@ -102,3 +102,25 @@ diagnostics:
         script: weighting/weighted_temperature_graph.py
         ancestors: [calculate_weights_climwip/climwip, tas]
         weights: 'weights.nc'
+
+  weighted_temperature_map:
+    variables:
+      tas: &map_settings
+        short_name: tas
+        start_year: 2081
+        end_year: 2100
+        mip: Amon
+        preprocessor: climatological_mean
+      tas_reference:
```

Comment:
It seems counterintuitive to me to use the anomalies preprocessor here. To me this is not an anomaly but a temperature change between two time periods. I agree that it would be nice to do as much as possible in the preprocessor, but it would be more intuitive to calculate the two climatologies in the recipe and subtract them from each other.
I need something similar here #1850 and opened an issue here https://github.com/ESMValGroup/ESMValCore/issues/833

Reply:
I changed this up a bit. I think it is more flexible to use two climatologies here and calculate the difference in the script. This way we have the option of skipping the second climatology, in which case we plot absolute temperature instead of temperature change.